workspaces/simon-game/src/app/components/App.tsx

```diff
@@ -1,8 +1,26 @@
 import React from "react";
+import { buttonSoundConfig } from "../constants.ts";
+import playSound from "../util/playSound.ts";
 
 export function App() {
+  let audioContext: AudioContext | null = null;
```

Comment:
If you actually want to cache this, it has to go outside the React component, otherwise it will be recreated on every render.

Reply:
ack!